src/estimation/include/iDynTree/Estimation/AttitudeQuaternionEKF.h

```diff
@@ -76,7 +76,7 @@ namespace iDynTree
      * @note: we will drop the subscripts and superscripts in the rest of the documentation for convenience
      *
      * Discretized dynamics during the prediction step,
-     * \f[ \hat{{x}}_{k+1} = \begin{bmatrix} q_k + \Delta t. \frac{1}{2}q_k  \circ \begin{bmatrix} 0 \\ \omega^b_{k+1} \end{bmatrix} \\ y_{gyro_{k}} - b_{k} \\ (1 - \lambda_{b} \Delta t)b_k \end{bmatrix} \f]
```

Comment:
@traversaro I changed also here the quaternion propagation. Do you think this will affect the corresponding Jacobian? Also, I am doubting whether our implementation is linearizing around the correct operating point. ;D